### PR TITLE
Improve the integration of the Metrics pipeline in the release

### PR DESCRIPTION
**Pull request by ireneisdoomed**
This PR includes:
- The changes to adapt the metrics script to work with remote data ([#2613](https://github.com/opentargets/issues/issues/2613))
- The changes to parametrise the metrics script with a configuration file ([#2614](https://github.com/opentargets/issues/issues/2614))
- General repo restructure to make it cleaner

### CODE REVIEW

File: docs/metric-calculation.md [hunk 1]
```diff
@@ -1,141 +1,41 @@
...
-  --gold-standard-associations gold-standard/informa_abbvie.tsv \
-  --gold-standard-mappings gold-standard/mesh_mappings.tsv
+# Package the repo structure to provision the cluster with it
+zip -x src/metric_calculation/metrics.py -r code_bundle.zip .
+
+gcloud dataproc jobs submit pyspark \
+  src/metric_calculation/metrics.py \
+  --cluster=${CLUSTER_NAME} \
+  --py-files code_bundle.zip \
+  --files='config/config.yaml'
```
Comment (DSuveges) [inline docs/metric-calculation.md:42]: Region is required to submit.

```suggestion
  --region=${CLUSTER_REGION} \
  --files='config/config.yaml'
```

File: docs/metric-calculation.md [hunk 2]
```diff
@@ -1,141 +1,41 @@
...
-  ${ETL_PARQUET_OUTPUT_ROOT}/targets \
-  ${ETL_PARQUET_OUTPUT_ROOT}/molecule \
-  post-pipeline
-gsutil -m cp -r \
-  ${ETL_PARQUET_OUTPUT_ROOT}/errors/evidence \
-  post-pipeline/evidenceFailed
+export CLUSTER_NAME=ot-release-metrics
+export CLUSTER_REGION=europe-west1
+
+gcloud dataproc clusters create ${CLUSTER_NAME} \
```
Comment (DSuveges) [inline docs/metric-calculation.md:21]: This command create a cluster with a master and two workers. Is there a reason not to use a single node dataproc cluster?

**Review by DSuveges — CHANGES_REQUESTED** [review]
I feel there's somethign off, we need to optimize probably the creation of the machine or the job submission.

File: docs/metric-calculation.md (hunk 2 above)
Comment (ireneisdoomed) [reply docs/metric-calculation.md:21]: Thanks, I'd simply forgotten adding the flag.

**Comment by ireneisdoomed** [comment]
I've been able to run the metrics script in ~20min for the first 22.09 pipeline run following the instructions in the README. Logs available at https://console.cloud.google.com/dataproc/jobs/ac69daf526dd4c6091b73e48e0fa2719/monitoring?region=europe-west1&project=open-targets-eu-dev

**Pull request merged**